notebooks/samples/explanations/ai-explanations-image-tf2.ipynb

```diff
@@ -0,0 +1,1758 @@
+{
```

Comment:
<p>Otherwise, set your project ID here:</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/ai-platform-samples/pull/192/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>